tfjs-backend-wasm/src/setup_test.ts

```diff
@@ -396,6 +390,7 @@ const TEST_FILTERS: TestFilter[] = [
   {include: 'asinh '},
   {include: 'diag '},
   {include: 'denseBincount '},
+  {include: 'bitwiseAnd'},
```

Comment:
Now are you sure that the test is actually running? A quick way I usually do is intentionally making a wrong implementation (like `return a & b` -> `return a ^ b`) and see if the tests can fail. If so I can confidently tell that the tests are working.

Reply:
Thanks, the test is actually running.